internal/configs/configload/testing.go

```diff
@@ -23,11 +22,12 @@ import (
 func NewLoaderForTests(t *testing.T) (*Loader, func()) {
 	t.Helper()
 
-	modulesDir, err := ioutil.TempDir("", "tf-configs")
+	modulesDirDescriptor, err := os.CreateTemp("", "tf-configs")
```

Comment:
I think you're replacing a call to create a directory with a call to [create a file](https://pkg.go.dev/os#CreateTemp).

Reply:
You're right, I've changed it.